disdat/apply.py

```diff
@@ -51,7 +51,7 @@ def apply(output_bundle, pipe_params, pipe_cls, input_tags, output_tags, force,
     Args:
         output_bundle: The new bundle to be created
         pipe_params (str):   Luigi Task parameters string
-        pipe_cls:      String <module.ClassName>
+        pipe_cls (type[PipeTask]):      reference to the task class
```

Comment:
So I know we can now declare types in P3.   But I haven't seen this syntax for types in google doc strings.   I usually say `disdat.pipe.PipeTask` for the type.

Reply:
i can change it to that, but technically the (type) in the parentheses is the thing returned by type(value).  by putting disdat.pipe.PipeTask, you are implying that the value should be an INSTANCE of that class.  

this type[class] notation is understood by pycharm, so i believe it is standard.

perhaps it should be type[disdat.pipe.PipeTask] to be more explicit.

thoughts?

Reply:
Got it.   The small change then to the fully qualified type[disdat.pipe.PipeTask] sounds perfect.